app/models/support/service_performance.rb

```diff
@@ -1,4 +1,318 @@
 class Support::ServicePerformance
+  def percentage_of_devolved_schools_that_have_signed_in
+    if number_of_devolved_schools.positive?
+      (number_of_devolved_schools_that_have_signed_in * 100.0 / number_of_devolved_schools).round
+    else
+      0
+    end
+  end
+
+  def number_of_devolved_schools_that_have_signed_in
+    @number_of_devolved_schools_that_have_signed_in ||=
+      School
+      .gias_status_open
+      .that_will_order_devices
+      .joins(user_schools: :user)
+      .where('users.sign_in_count > 0')
+      .count('DISTINCT(schools.id)')
+  end
+
+  def number_of_devolved_schools
+    @number_of_devolved_schools ||=
+      School
+      .gias_status_open
+      .that_will_order_devices
+      .count
+  end
+
+  def number_of_centrally_managed_schools
+    @number_of_centrally_managed_schools ||=
+      School
+      .gias_status_open
+      .that_are_centrally_managed
+      .count
+  end
+
+  def total_devices_available
+    SchoolDeviceAllocation.std_device.sum(:allocation)
```

Comment:
Would using `cap` here and elsewhere in this file be marginally safer?

Reply:
It could be, I don't know tbh, I thought `allocation` was the 'real' figure and `cap` just how much of that was available now but it has become a bit blurred?

Reply:
Yeah, I'm not sure what the best option is here. The difference between cap and devices ordered is what you can order now, not allocation and devices ordered...

Reply:
The total for the blue box should equal the sum of the green and orange boxes below it